test/k8sT/Policies.go

```diff
@@ -131,6 +135,72 @@ var _ = Describe("K8sPolicyTest", func() {
 				"Policy status does not match for endpoint %s", appPods[helpers.App1])
 		}
 
+		importPolicy := func(file string) {
+			_, err := kubectl.CiliumPolicyAction(
+				helpers.KubeSystemNamespace, file, helpers.KubectlApply, helpers.HelperTimeout)
+			ExpectWithOffset(1, err).Should(BeNil(),
+				"policy %s cannot be applied in %q namespace", file, helpers.DefaultNamespace)
+		}
+
+		validatePolicyEnforcementStatus := func(desiredState models.EndpointPolicyEnabled) {
+			ExpectWithOffset(1, kubectl.WaitCEPReady()).To(BeNil(), "Cep is not ready after a specified timeout")
+			for _, app := range []string{helpers.App1, helpers.App2, helpers.App3} {
+				pod := appPods[app]
+				cep := kubectl.CepGet(helpers.DefaultNamespace, pod)
+				ExpectWithOffset(1, cep).NotTo(BeNil(), "Endpoint %q does not exist", pod)
+
+				ExpectWithOffset(1, cep.Status.Policy.Realized.PolicyEnabled).To(
+					Equal(desiredState),
+					"Unexpected policy enforcement status for endpoint %s. have: %v want: %v",
+					pod, cep.Status.Policy.Realized.PolicyEnabled, desiredState)
+			}
+		}
+
+		validateConnectivity := func(expectWorldSuccess, expectClusterSuccess bool) {
+			for _, pod := range []string{appPods[helpers.App2], appPods[helpers.App3]} {
+				res := kubectl.ExecPodCmd(
+					helpers.DefaultNamespace, pod,
+					helpers.CurlFail("http://www.google.com/"))
+
```

Comment:
Would be helpful to add calls to `By` here that contain the values of `expectWorldSuccess` and `expectClusterSuccess`. It will make the test more readable / easier to follow.